crates/autopilot/src/infra/persistence/dto/auction.rs

```diff
@@ -33,34 +33,11 @@ pub fn from_domain(auction: domain::Auction) -> Auction {
     }
 }
 
-pub fn try_to_domain(auction: Auction) -> anyhow::Result<domain::Auction> {
-    Ok(domain::Auction {
-        block: auction.block,
-        latest_settlement_block: auction.latest_settlement_block,
-        orders: auction
-            .orders
-            .into_iter()
-            .map(super::order::to_domain)
-            .collect(),
-        prices: auction
-            .prices
-            .into_iter()
-            .map(|(key, value)| {
-                Price::new(value.into()).map(|price| (eth::TokenAddress(key), price))
-            })
-            .collect::<Result<_, _>>()?,
-        surplus_capturing_jit_order_owners: auction
-            .surplus_capturing_jit_order_owners
-            .into_iter()
-            .map(Into::into)
-            .collect(),
-    })
-}
-
 #[serde_as]
 #[derive(Clone, Debug, Serialize, Deserialize)]
+#[serde(rename = "Auction")]
```

Comment:
I am not sure if this is safe to just rename, so just did a safe rename by default. If we don't want to risk it, I guess we can merge the PR as is, and then rename in a separate PR just in case we need to quickly revert. 
But also fine with renaming immediately, as any issue would probably be caught on staing.

Reply:
Yeah, I'm 99% sure the struct name is irrelevant when we use JSON. This could be different for other serialization formats but with JSON we should be fine. So let's not introduce things people in the future might think have to be kept like this to keep compatibility.

Reply:
removed